hardware/sensor/lywsd03mmc_sensor.py

```diff
@@ -0,0 +1,32 @@
+from . import terrariumBluetoothSensor
+
+# pip install lywsd03mmc
+from lywsd03mmc import Lywsd03mmcClient
+
+class terrariumLywsd03mmcSensor(terrariumBluetoothSensor):
+  HARDWARE = 'LYWSD03MMC'
+  TYPES    = ['temperature','humidity']
+  NAME     = 'LYWSD03MMC bluetooth sensor'
+
+  def _load_hardware(self):
+    address = self._address
+    device = Lywsd03mmcClient(address[0])
+    return device
+
+  def _get_data(self):
+    data = {}
+
+    # BUG: Firmware data en battery will only be set at the first measurement.
+    #self._device['firmware'] = self._device['device'].firmware_version()
+    #self._device['battery']  = self._device['device'].parameter_value(MI_BATTERY)
+
+    data['temperature'] = self.device.temperature
+    data['humidity']    = self.device.humidity
+    data['battery']     = self.device.battery
+
+    return data
+
+  @staticmethod
+  def _scan_sensors(unit_value_callback = None, trigger_callback = None):
+    for sensor in terrariumBluetoothSensor._scan_sensors(__class__, ['mj_ht_v1'], unit_value_callback, trigger_callback):
```

Comment:
This is actually leftovers from my first pass at this, it actually uses "lywsd03mmc" and this seems to work okay for me.

I'm going to test this on my "production" setup in the next couple days.

Reply:
I seem to be having problems getting my containerized terrariumpi setup to scan bluetooth correctly.

I think the code is right, I just need to figure out my scanning problems.

Reply:
Do you have errors, or are the devices not found. If not found, then try to add some debugging:
> You can test this, by adding a print line after https://github.com/theyosh/TerrariumPI/blob/main/hardware/sensor/__init__.py#L453 and print the value device.getValueText(9) if that gives a different string then 'mj_ht_v1' then we can use that.

Not sure if a print line will work in Docker, as I develop natively on the Pi and not in Docker.

Because I can just scan bluetooth devices in my docker setup. So I think that should be working. I am guessing that these different MiTemp sensors have a different name then `mj_ht_v1`

Reply:
I figured out my scanning problem, it really does need the NET_ADMIN capability. It would sort of find them, but then not be able to really connect.

The name they advertise is `LYWSD03MMC` so we won't have conflicts with the other mitemp sensors named `mj_ht_v1`.